feature-utils/poly-look/src/react-components/overlays/sideSheet.css

```diff
@@ -0,0 +1,34 @@
+.poly-side-sheet {
+  max-width: 288px;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  background-color: var(--color-gray-100);
+  color: var(--color-dark);
+  z-index: 100;
```

Comment:
We might need to do something about z-index, up to an including a central definition for it. As I understand this, this should be always on top. However, there are components with indices up to 400, and some that have the same value of 100. 
![Captura de pantalla de 2022-04-28 07-44-36](https://user-images.githubusercontent.com/500/165685136-873ab6b8-0b5a-42b4-8ece-df6afb383d44.png)
Most popups share the same index, 100. There are many possible combinations, so we need either a way to test how components combine, or an unified approach to assigning indices to components (base:0 popups: 20 sidesheets:40, for instance) or maybe both.
Within the scope of this PR, as you see the highest "space odyssey" index is 2001, so you should maybe go for "marvel" 2099, in the off-chance that sidesheet needs to go over scrollable.

Reply:
Makes sense (I just love `z-index`, it's almost impossible to not use it super wonkily with lots of arbitrary values over time :D)

Reply:
Honestly, I was tempted to go for 9999 😂  That's the value I'm mostly used to when it comes to pop-ups. However that was leftover css that I didn't even noticed and it's not needed here.  If things don't go well, it will have to be changed in `SideSwiper`.